File: Task2/apz-pzpi-21-11-milennyi-ivan-task2/src/view_models/breed_vm.rs
use serde::Serialize;
use sqlx::FromRow;
use utoipa::ToSchema;

#[derive(Debug, Serialize, FromRow, ToSchema)]
pub(crate) struct BreedVM {
    id: u64,
    name: String,
    info: String,
    feed_name: String,
    sheep_count: u64,
}

File: Task4/apz-pzpi-21-11-milennyi-ivan-task4/src/viewModels/BreedVM.ts
export class BreedVM {
    id: number;
    name: string;
    info: string;
    feedName: string;
    sheepCount: number;

    constructor(id: number, name: string, info: string, feedName: string, sheepCount: number) {
        this.id = id;
        this.name = name;
        this.info = info;
        this.feedName = feedName;
        this.sheepCount = sheepCount;
    }
}